kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -211,7 +212,14 @@ impl<'tcx> GotocCtx<'tcx> {
                 }
                 Some(sig)
             }
-            ty::Generator(_def_id, _substs, _movability) => None,
+            ty::Generator(_def_id, _substs, _movability) => {
+                let error_msg = format!(
+                    "Crate `{}` uses generators which is currently not supported by Kani",
+                    self.short_crate_name()
+                );
+                self.tcx.sess.err(&error_msg);
+                FatalError.raise()
```

Comment:
Haven't we written a standard function for these errors? `unsupported_msg` seems to be building a message similar to this one, but I don't see any function for erroring out.

Reply:
Good point. Done.